tests/baselines/reference/strictOptionalProperties1.js

```diff
@@ -353,103 +377,14 @@ a = e; // Error
 
 
 //// [strictOptionalProperties1.d.ts]
-declare function f1(obj: {
-    a?: string;
-    b?: string | undefined;
-}): void;
-declare function f2(obj: {
-    a?: string;
-    b?: string | undefined;
-}): void;
-declare function f3(obj: Partial<{
-    a: string;
-    b: string | undefined;
-}>): void;
-declare function f4(t: [string?]): void;
-declare function f4a(t1: [number, string?], t2: [number, string?, string?]): void;
-declare function f5(t: [number, string?, boolean?]): void;
-declare function f6(): void;
-declare type Props = {
-    foo: string;
-    bar: string;
-};
-declare type InputProps = {
-    foo?: string;
-    bar: string;
-};
-declare const defaultProps: Pick<Props, 'foo'>;
-declare const inputProps: InputProps;
-declare const completeProps: Props;
-declare const t1: [number, string?, boolean?];
-declare const t2: [number, string?, boolean?];
-declare const t3: [number, string?, boolean?];
-declare const t4: [number, string?, boolean?];
-declare const x: {
-    foo?: number;
-};
-declare const y: {
-    foo: number;
-};
-interface Test {
-    [key: string]: string;
-    foo?: string;
-    bar?: string | undefined;
-}
-declare let ox1: {
-    p: string;
-};
-declare let ox2: {
-    p: string | undefined;
-};
-declare let ox3: {
-    p?: string;
-};
-declare let ox4: {
-    p?: string | undefined;
-};
-declare let tx1: [string];
-declare let tx2: [string | undefined];
-declare let tx3: [string?];
-declare let tx4: [(string | undefined)?];
-declare function f11<T>(x: {
-    p?: T;
-}): T;
-declare function f12<T>(x: [T?]): T;
-declare function f13<T>(x: Partial<T>): T;
-declare type Undefinable<T> = T | undefined;
-declare function expectNotUndefined<T>(value: Undefinable<T>): T;
-interface Bar {
-    bar?: number;

```

Comment:
Why did all this disappear from the declaration emit baseline when you only added to the test? I feel like GitHub must be trolling me or something...

Reply:
Oh, I see, it's because of the `export` specifiers turning the whole file into a module, and therefore *not* exporting all of the other stuff. I'll get rid of those.